pytest.ini

```diff
@@ -1,2 +1,4 @@
 [pytest]
 asyncio_mode = auto
+log_cli = true
+log_cli_level = INFO
```

Comment:
nice, for test config, let's use DEBUG since we're changing most of them to `logging.debug`